### PR TITLE
Add empirical vs reported graph

### PR DESCRIPTION
**Pull request by sstadick**
This PR adds adds support for generating an empirical vs reported quality graph to the GATK Base Recalibrator module. 

 - [X] This comment contains a description of changes (with reason)
 - [X] `CHANGELOG.md` has been updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,8 @@
     * Log10 x-axis for _Read Length_ plot ([#1214](https://github.com/ewels/MultiQC/issues/1214))
 * **fgbio**
     * Fix `ErrorRateByReadPosition` to calculate `ymax` not just on the overall `error_rate`, but also specific base errors (ex. `a_to_c_error_rate`, `a_to_g_error_rate`, ...).  ([#1215](https://github.com/ewels/MultiQC/pull/1251))
+* **GATK**
+  * Add support for creation of a "Reported vs Empirical Quality" graph to the Base Recalibration module.
```
Comment (nh13) [inline CHANGELOG.md:31]:
```suggestion
  * Add support for the creation of a "Reported vs Empirical Quality" graph to the Base Recalibration module.
```

File: multiqc/modules/gatk/base_recalibrator.py [hunk 2]
```diff
@@ -137,6 +140,61 @@ def add_quality_score_vs_no_of_observations_section(self):
...
+                    try:
+                        reported_empirical[sample].append(
+                            {
+                                'x': int(reported),
+                                'y': sum(float(r['EmpiricalQuality']) for r in g) / len(g),
+                            }
+                        )
+                    except ZeroDivisionError:
+                        reported_empirical[sample][int(reported)] = 0
+                        reported_empirical[sample].append({'x': int(reported), 'y': 0})
```
Comment (nh13) [inline multiqc/modules/gatk/base_recalibrator.py:171]: I know in Python were supposed to "ask for forgiveness", but I think this is clearer.
```suggestion
                    reported_empirical[sample].append(
                        {
                            'x': int(reported),
                            'y': sum(float(r['EmpiricalQuality']) for r in g) / len(g) if len(g) > 0 else 0,
                        }
                    )
```

**Review by nh13 — APPROVED** [review]
Two minor nits but LGTM.  @ewels I think this one is ready for your review.

**Comment by sstadick** [comment]
Thanks for the suggestions @nh13!

@ewels let me know if there are further changes you'd like to see as well!

**Comment by sstadick** [comment]
@ewels, just wanted to check in and see if you've had a chance to look at this!

**Review by ewels — APPROVED** [review]
Tests passing, haven't tried the code myself but looks fine on a quick scan 👍🏻 

Thanks!

**Comment by sstadick** [comment]
Thanks @ewels!

**Pull request merged**